openssl/src/ssl/mod.rs

```diff
@@ -690,6 +691,24 @@ impl Ssl {
         Ok(ssl)
     }
 
+    pub fn get_state_string<'a>(&self) -> &'a str {
+        let state = unsafe {
```

Comment:
The lifetime isn't static, so an unbound lifetiem should be appropriate:

    const char *SSL_state_string(const SSL *ssl);
    const char *SSL_state_string_long(const SSL *ssl);